py/kubeflow/kubeflow/ci/workflow_utils.py

```diff
@@ -0,0 +1,249 @@
+import logging
+import os
+
+from kubeflow.testing import argo_build_util
+
+# The name of the NFS volume claim to use for test files.
+NFS_VOLUME_CLAIM = "nfs-external"
+# The name to use for the volume to use to contain test data
+DATA_VOLUME = "kubeflow-test-volume"
+
+E2E_DAG_NAME = "e2e"
+EXIT_DAG_NAME = "exit-handler"
+
+
+LOCAL_TESTING = os.getenv("LOCAL_TESTING", "False")
+CREDENTIALS_VOLUME = {"name": "aws-secret",
+                      "secret": {"secretName": "aws-secret"}}
+CREDENTIALS_MOUNT = {"mountPath": "/root/.aws/",
+                     "name": "aws-secret"}
+
+AWS_WORKER_IMAGE = ("527798164940.dkr.ecr.us-west-2.amazonaws.com/"
```

Comment:
We now migrated to new test-infra, so the worker image should be `public.ecr.aws/j1r0q0g6/kubeflow-testing:latest`

Reply:
ACK, I'll use the public one